lib/KTextbox/__tests__/KTextbox.spec.js

```diff
@@ -0,0 +1,158 @@
+import { shallowMount, mount } from '@vue/test-utils';
+import KTextbox from '../index';
+
+describe('KTextbox component', () => {
+  it(`smoke test`, () => {
+    const wrapper = shallowMount(KTextbox);
+    expect(wrapper.exists()).toBe(true);
+  });
+
+  describe('props', () => {
+    it(`a label should appear`, () => {
+      const wrapper = mount(KTextbox, {
+        propsData: {
+          label: 'test',
+        },
+      });
+      expect(wrapper.find('label').text()).toEqual('test');
+    });
+    it(`value of the text field should appear when passed in`, () => {
+      const initialValue = '1234paowiejfapwoeifjapwoeijf';
+      const wrapper = mount(KTextbox, {
+        propsData: {
+          label: 'test',
+          value: initialValue,
+        },
+      });
+      const value = wrapper.find('input').element.value;
+      expect(value).toBe(initialValue);
+    });
+    it(`invalidText is not displayed if showInvalidText is false`, () => {
+      const wrapper = mount(KTextbox, {
+        propsData: {
+          invalid: true,
+          showInvalidText: false,
+          invalidText: 'error!',
+        },
+      });
+      const errorTextField = wrapper.find('.ui-textbox-feedback-text');
+      expect(errorTextField.text()).not.toBe('error!');
+    });
+    it(`invalidText is displayed through error prop if invalid and showInvalidText are both true`, () => {
+      const wrapper = mount(KTextbox, {
+        propsData: {
+          invalid: true,
+          showInvalidText: true,
+          invalidText: 'error!',
+        },
+      });
+      const errorTextField = wrapper.find('.ui-textbox-feedback-text');
+      expect(errorTextField.text()).toBe('error!');
+    });
```

Comment:
I'm so sorry for this weird set of props for handling invalid states but glad to have tests added for it!